tvx/exec_lotus.go

```diff
@@ -29,38 +30,66 @@ var execLotusCmd = &cli.Command{
 		&cli.StringFlag{
 			Name:        "file",
 			Usage:       "input file",
-			Required:    true,
 			Destination: &execLotusFlags.file,
 		},
+		&cli.BoolFlag{
+			Name:        "stdin",
+			Usage:       "",
+			Destination: &execLotusFlags.stdin,
+		},
 	},
 	Action: runExecLotus,
 }
 
 func runExecLotus(_ *cli.Context) error {
-	if execLotusFlags.file == "" {
-		return fmt.Errorf("test vector file cannot be empty")
-	}
+	switch {
```

Comment:
We probably want to XOR validate the different input modes, i.e. user can't provide many input modes at once.